test/integration/bpmnlint-plugin-exported/index.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  configs: {
+    recommended: {
+      rules: {
+        'foo-exported-string': 'error',
+        'bar-exported-function': 'error'
+      }
+    }
+  },
+  rules: {
```

Comment:
We want to add additional rules here to verify if/how tree-shaking works. This will allow us to make a more profound decision on which mechanism (string referencing vs. require) to support.

Reply:
I can pick that up, try that.

Reply:
Picking up now.

Reply:
Okay, thank you!